### DISCUSSION TITLE
系统日记

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#40] [Q&A]
我的系统日记，感觉会自动删除。我想一直保存。这么设置
![image](https://github.com/densen2014/docs/assets/122285019/de358b0a-7469-4e75-8ec8-3f6e52b77fe4)

**Comment by densen2014** [comment]
你的感觉是错误的。不会自动删除日志

**Comment by densen2014** [comment]
明明是你自己点的清空日志，第一行写的
